### DISCUSSION TITLE
Filter entries by field values

### DISCUSSION THREAD

**Discussion by bengreenbank** [craftcms/cms#10230] [Ideas] [answered]
Had a quick look and couldn't see a similar request anywhere.

There have been a few instances where it would be useful to be able to filter entries by their field's value in the CMS. The sorting currrently works well, but would be good to extend on that with the ability to filter.

**Comment by brandonkelly** [comment, marked as answer]
Already added for Craft 4 – see #10075 :)

**Reply by bengreenbank** [reply]
Great, thank you!
